components/TARGET_PSA/services/attestation/COMPONENT_PSA_SRV_IMPL/attest_crypto.c

```diff
@@ -0,0 +1,277 @@
+/*
+* Copyright (c) 2018-2019 ARM Limited. All rights reserved.
+*
+* SPDX-License-Identifier: Apache-2.0
```

Comment:
Pls check whether is it OK from legal point of view to replace the license header in this file. It seems that this file is derived from TF_M repo ./lib/t_cose/src/t_cose_psa_crypto.c

Reply:
@moranpeker Please review